src/cryojax/core/_filtered_transformations.py

```diff
@@ -9,7 +10,7 @@
 from typing import Callable, Union, Any, Optional, Hashable
 
 
-def filter_grad(
+def filter_grad_with_spec(
```

Comment:
do you want to give a bit of orientation to these functions with a doctstring?

Reply:
yes good idea... I may do this in a future PR because in the next few weeks I'll be writing up some examples using this in practice. right now `cryojax.core` is mostly experimental and is lightweight extensions to `equinox`. I'm excited to write these up---it breaks down `equinox` into its intuitive components and will draw people in to learn more :).